### PR TITLE
Expand tests for coverage and CI

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: run-coverage [hunk 1]
```diff
@@ -11,8 +11,8 @@ except ImportError:
     exit(1)"
 
 rm -rf .coverage*
-PATH=$PWD:$PATH $COVERAGE run -p --include=$PWD/**/* ./tests/run-unit
-PATH=$PWD:$PATH RUNNER="$COVERAGE run -p --include=$PWD/**/*" ./run-test-ci
+PATH=$PWD:$PATH $COVERAGE run -p --include=$PWD/ocropus-*,$PWD/**/* ./tests/run-unit
```
Comment (kba) [inline run-coverage]: That seems redundant, `$PWD/**/*` glob should include `$PWD/ocropus-*`

Comment (zuphilip) [reply run-coverage]: Did you try it out?

Comment (kba) [reply run-coverage]: Yes, it works with just `$PWD/**/*` for me. Does it not for you?

File: run-test-ci [hunk 2]
```diff
@@ -13,36 +13,99 @@ RUNNER="${RUNNER:-python2}"
...
+
 rm -rf temp
+mkdir -p temp
 test_page
 test_conf
 test_linegen
 test_rtrain
+test_nlbin
+#this depends on #220:
+#test_gpageseg
```
Comment (zuphilip) [inline run-test-ci]: After merging #220 we should uncomment this line.

File: run-coverage (hunk 1 above)
Comment (zuphilip) [reply run-coverage]: I can't remember but I believe you. Feel free to change it then. A merge here is overdue.

**Comment by kba** [comment]
Works, https://travis-ci.org/tmbdev/ocropy/builds/281306698?utm_source=github_status&utm_medium=notification, thank you @zuphilip .

**Comment by kba** [comment]
Merged

**Pull request closed without merging**